rules/utils.go

```diff
@@ -38,12 +38,16 @@ var validMachineSizes = []string{
 	"Standard_D1",
 	"Standard_D11",
 	"Standard_D11_v2",
+	"Standard_D11_v2_Promo",
```

Comment:
The Promo series certainly seems to exist, but it doesn't seem to be mentioned anywhere in the official documentation. Where are these sources?
https://learn.microsoft.com/en-us/azure/virtual-machines/sizes